docs/getting_started.md

```diff
@@ -240,7 +240,7 @@ Arguments:
 |----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | SPHINX_SOURCE  | The root of your sphinx source/input directory tree.<br/>Often this is the same directory your conf.py is in.                                                               |
 | SPHINX_OUTPUT  | The root of you sphinx output directory tree - where sphinx puts the generated html files to.<br/> This should be the directory where sphinx put's it's main index.html to. |
-| DOXYFILE(S)    | One or many doxygen configuration files that all should stick to the setting recommendation from [STEP-3](#step-3)                                                          |
+| INPUT(S)    | One or many inputs where each input could be either...<ul><li>a doxygen configuration file (doxyfile). This is recommended for "beginners" because it will also check the config for doxysphinx compatibility.</li><li>an output path where the generated doxygen documentation resides. This is more like an "expert"-mode which is especially useful when integrating doxysphinx with buildsystems like cmake etc. which are dynamically generating doxygen configs.</li></ul>
```

Comment:
> the generated doxygen documentation resides

@mb-emag  Should it be?
> the generated doxygen config resides

Reply:
@twodrops - no it's the directory where doxygen generates the html files to (I wish i had a clearly defined name for that path) - This "expert" mode is for build-tools where the doxygen config is automatically generated (like cmake) and shouldn't be parsed by doxysphinx because the output path is known beforehand by the build-tool (e.g. with a cmake variable). It's also a fallback for very complex doxyfiles (doxygen configs) where our parser wouldn't be capable of getting the right output path.